vue-frontend/src/components/home-new/HomeIndex.vue

```diff
@@ -62,6 +62,7 @@ export default {
   },
   mounted() {
     this.$gtag.event("view_landing_section");
+    document.addEventListener("touchstart", handler, { passive: true });
```

Comment:
What is purpose of it?

Reply:
To solve this "Does not use passive listeners to improve scrolling performance" . I found this only solution for it.

Reply:
Shouldn't we need to pass our defined handler in `handler`?

Reply:
Passive listeners do not work on scroll events.